filter-50-invoked.conf

```diff
@@ -0,0 +1,17 @@
+filter {
+  # remove if if really *every* message starts like this
+  if [message] =~ /Invoked with/ {
+
+    grok {
+      match => ["message", "Invoked with %{GREEDYDATA:[@metadata][ansiblekv]}"]
```

Comment:
what does the @ do?

Reply:
I think you store what comes after "invoked with" in **ansiblekv** then you wright this in a field name **ansible** is this right?

Reply:
Right. I'm using a neat little trick: I'm writing into field `@metadata.ansiblekv`. Every field beneath `@metadata` is not written into outdputs. This way I safe an extra step for deleting a temporary field. It just get's deleted. I just pushed another change where I use this field as source for `kv`.